module-system/module-implementations/sov-bank/src/call.rs

```diff
@@ -263,7 +263,7 @@ impl<C: sov_modules_api::Context> Bank<C> {
     pub fn get_token_name(
         &self,
         token_address: &C::Address,
-        working_set: &mut WorkingSet<C>,
+        working_set: &mut impl StateMapWorkingSet,
```

Comment:
ah, so not entirely (and not in all places, primarily query.rs).
but that's a good point since gas should be charged for cross-module calls, otherwise it will prevent module1 from calling module2. will add a gas trait for that an impl it for regular working set - will check if that is feasible

Reply:
Q: What was the reason for `impl StateMapWorkingSet` instead a concrete type?